addons/mail/static/src/js/services/chat_manager.js

```diff
@@ -1115,20 +1114,93 @@ var ChatManager =  AbstractService.extend({
             self.mentionPartnerSuggestions = result.mention_partner_suggestions;
             self.discussMenuID = result.menu_id;
 
-            // Shortcodes: canned responses and emojis
+            // Shortcodes: canned responses            
             _.each(result.shortcodes, function (s) {
-                if (s.shortcode_type === 'text') {
-                    self.cannedResponses.push(_.pick(s, ['id', 'source', 'substitution']));
-                } else {
-                    self.emojis.push(
-                        _.pick(s, ['id', 'source', 'unicode_source', 'substitution', 'description'])
-                    );
-                    self.emojiSubstitutions[_.escape(s.source)] = s.substitution;
-                    if (s.unicode_source) {
-                        self.emojiSubstitutions[_.escape(s.unicode_source)] = s.substitution;
-                        self.emojiUnicodes[_.escape(s.source)] = s.unicode_source;
-                    }
-                }
+                self.cannedResponses.push(_.pick(s, ['id', 'source', 'substitution']));
+            });
+            // emojis
+            self.emojiUnicodes={
+                ":)":"😊",
+                ":D":"😃",
+                "xD":"😆",
+                ":-)":"😂",
+                ";)":"😉",
+                "B)":"😎",
+                ";p":"😜",
+                ":p":"😋",
+                "xp":"😝",
+                "o_o":"😳",
+                ":|":"😐",
+                ":/":"😕",
+                ":(":"😞",
+                ":@":"😱",
+                ":O":"😲",
+                ":'o":"😨",  
+                "3:":"😠",
+                "3:)":"😈",
```

Comment:
`>:(` and `>:)`

Reply:
As an alternative solution yes but 3:) is the facebook shortcut for the devil.

Reply:
Damn Zuckerberg!! :angry:

Reply:
As far as I remember it was the same on msn messenger :D

Reply:
MSN, wow where have the times gone too. Now you made me nostalgic! 3: